app/controllers/admin_general_controller.rb

```diff
@@ -52,9 +52,14 @@ def index
                          @body_update_requests ].
       any?{ |to_do_list| ! to_do_list.empty? }
 
-    @attention_comments = Comment.
-      where(:attention_requested => true).
-        not_embargoed
+    # HACK: Running this query through ActiveRecord freezes…
+    attention_comments_sql = Comment.connection.unprepared_statement do
```

Comment:
Doesn't look like we need the `unprepared_statement` block actually. I'm convinced I needed this yesterday because [`to_sql` displays `$1` instead of real value after upgrade to Rails 4.1.1](https://github.com/rails/rails/issues/15331).